README.md

```diff
@@ -497,6 +496,20 @@ docker run -p 8080:80 \
    --max-total-tokens 8192 --max-batch-total-tokens 32768
 ```
 
+## TGI-Gaudi Benchmark
+
+### Static Batching Benchmark
+ To run static batching benchmark, please refer to [TGI's benchmark tool](https://github.com/huggingface/text-generation-inference/tree/main/benchmark).
+
+   To run it on the same machine, you can do the following:
+   * `docker exec -it <docker name> bash` , pick the docker started from step 2 using docker ps
+   * `text-generation-benchmark -t <model-id>` , pass the model-id from docker run command
+   * after the completion of tests, hit ctrl+c to see the performance data summary.
```

Comment:
worth mentioning the benchmark configs like "this benchmark runs the model with bs=[1, 2, 4, 8, 16, 32],  prompt len=10, decode len=8 by default. if you want to run other configs, please check text-generation-benchmark -h and run with those params"

many customers didn't notice the benchmark configuration is different from the server configuration.

Reply:
added note